### PR TITLE
add challenge employees

### PR DESCRIPTION
**Pull request by Roland-Hufnagel**
### 🔎 Review

Review via [🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/js-array-methods-2_employees/sessions/js-array-methods-2/employees?file=/README.md) or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/js-array-methods-2_employees/sessions/js-array-methods-2/employees -i
```

### ✨ Copyable Markdown for Curriculum

````md
Solve this challenge on
[🔗 **CodeSandbox**](https://codesandbox.io/s/github/neuefische/web-exercises/tree/main/sessions/js-array-methods-2/employees?file=/README.md)
or locally by running this command in your Terminal:

```
npx ghcd@latest neuefische/web-exercises/tree/main/sessions/js-array-methods-2/employees -i
```
````

### CODE REVIEW

File: sessions/js-array-methods-2/employees/index.html [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  <body>
+    <h1>Human Ressources</h1>
+    <hr />
+    <div>
+      <form data-js="form1">
+        <label for="exercise1">Find by last name:</label>
+        <input type="text" id="exercise1" name="exercise1" />
+        <button>🔍 Search</button>
+      </form>
+      <div data-js="exercise1"></div>
```
Comment (moonwave99) [inline sessions/js-array-methods-2/employees/index.html:22]: If you want to be very picky, you can use the [`<output>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/output) element ^^

File: sessions/js-array-methods-2/employees/utils/db.js
Comment (moonwave99) [inline sessions/js-array-methods-2/employees/utils/db.js:1]: We can use a little smaller data sample I think : D

File: sessions/js-array-methods-2/employees/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
+# JS Array Methods 2: Employees
+
+This challenge tasks you with navigating a large dataset of over 500 employees, utilizing JavaScript array methods to find out if there is any employee that fits a specific condition.
+
+## Task
+
+Take a look at the browser preview. You will see large set of index cards, one card for each employee. There are four forms that help you finding a specific employees. Unfortunately the forms do not work correct. Your task is to implement the corresponding functions correctly. See `./js/index.js` for more information.
+
```
Comment (moonwave99) [inline sessions/js-array-methods-2/employees/README.md:8]: The forms work now! Have you pushed the solution accidentally perhaps? ^^

**Review by moonwave99 — CHANGES_REQUESTED** [review]
I thinks it's a very good exercise for advanced students, that want to see how to use array methods in an app context.

Let's address the few comments and add it to our roster!

File: sessions/js-array-methods-2/employees/utils/db.js
Comment (Roland-Hufnagel) [reply sessions/js-array-methods-2/employees/utils/db.js:1]: That's right. But this example with it's huge data shows the fantastic benefit of using array methods.

File: sessions/js-array-methods-2/employees/README.md (hunk 2 above)
Comment (Roland-Hufnagel) [reply sessions/js-array-methods-2/employees/README.md:8]: No - I just wanted to help you to find a solution 😂. Removed it now.

File: sessions/js-array-methods-2/employees/utils/db.js
Comment (moonwave99) [reply sessions/js-array-methods-2/employees/utils/db.js:1]: Trim it to 20 employees and we can merge it ^^

**Review by moonwave99 — APPROVED** [review]

**Pull request merged**